fastai/data/block.py

```diff
@@ -77,7 +77,15 @@ class DataBlock():
     blocks,dl_type = (TransformBlock,TransformBlock),TfmdDL
     _methods = 'get_items splitter get_y get_x'.split()
     _msg = "If you wanted to compose several transforms in your getter don't forget to wrap them in a `Pipeline`."
-    def __init__(self, blocks=None, dl_type=None, getters=None, n_inp=None, item_tfms=None, batch_tfms=None, **kwargs):
+    def __init__(self,
+        blocks:list=None, # List of `TransformBlock`s
+        dl_type:TfmdDL=None, # Specific class that inherits `TfmdDL`
```

Comment:
```suggestion
        dl_type:TfmdDL=None, # Task-specific `TfmdDL`, defaults to `block`'s dl_type or `TfmdDL` if none specified
```
Not sure I like my version, but tried to convey the hierarchy of how dl_type is populated: `__init__.dl_type` > `block.dl_type` > `self.dl_type`